x-pack/plugins/lens/public/xy_visualization/to_expression.ts

```diff
@@ -79,8 +79,10 @@ export const buildExpression = (
   metadata: Record<string, Record<string, OperationMetadata | null>>,
   frame?: FramePublicAPI
 ): Ast | null => {
-  const validLayers = state.layers.filter((layer): layer is ValidLayer =>
-    Boolean(layer.xAccessor && layer.accessors.length)
+  const validLayers = state.layers.filter(
+    (layer): layer is ValidLayer =>
+      !(layer.seriesType.includes('percentage') && !layer.splitAccessor) &&
```

Comment:
This just a stylistic suggestion, as that double negation was tricking me while reading:

```js
!layer.seriesType.includes('percentage') && 
  layer.splitAccessor && 
  Boolean(layer.accessors.length)
```